.github/analytics/get_repo_metrics.py

```diff
@@ -407,4 +408,4 @@ def main(
   plt.show()
 
 if __name__ == '__main__':
-  main()
+  typer.run(main)
```

Comment:
I believe that we use "from absl import app" and "app.main" in other places in the project to run the main function. Perhaps then we do not need the `typer` dependency.

Reply:
Let me try changing it to `absl` for consistency.

Reply:
Ok, update to use `absl`.